packages/trip-details/src/TripDetails.story.tsx

```diff
@@ -48,6 +54,52 @@ const StyledTripDetails = styled(TripDetails)`
   }
 `;
 
+const fareByLegLayout: FareTableLayout[] = [
+  {
+    header: "regular" as FareTableText,
```

Comment:
Thank you! I wasn't sure if this would be considered too verbose when something much simpler would do, but I thought it was worth doing to make the fixed set of values more obvious and easier to maintain in the future.